src/workers/notifications/utils.ts

```diff
@@ -54,11 +75,13 @@ export const getSubscribedMembers = (
       notificationType: type,
       referenceId,
       type: notificationPreferenceMap[type],
-      status: NotificationPreferenceStatus.Muted,
+      status: byStatus || byNotStatus,
```

Comment:
How are both under same where?

Reply:
I can split it into two functions instead

Reply:
let's do that please, I think it will be easier